cdp_scrapers/legistar_utils.py

```diff
@@ -867,20 +868,38 @@ def fix_event_minutes(
         """
         if not ev_minutes_item:
             return ev_minutes_item
-
+        rx = re.compile(r"\W+\s+")
         if ev_minutes_item.minutes_item and ev_minutes_item.matter:
             # we have both matter and minutes_item
             # - make minutes_item.name the more concise text e.g. "CB 11111"
             # - make minutes_item.description the more descriptive lengthy text
             #   e.g. "AN ORDINANCE related to the..."
             # - make matter.title the same descriptive lengthy text
             ev_minutes_item.minutes_item.description = ev_minutes_item.minutes_item.name
+            ev_minutes_item.minutes_item.description = rx.sub(
+                " ", str(ev_minutes_item.minutes_item.description)
+            ).strip()
             ev_minutes_item.minutes_item.name = ev_minutes_item.matter.name
+            ev_minutes_item.minutes_item.name = rx.sub(
+                " ", str(ev_minutes_item.minutes_item.name)
+            ).strip()
             ev_minutes_item.matter.title = ev_minutes_item.minutes_item.description
-
+            ev_minutes_item.matter.title = rx.sub(
+                " ", str(ev_minutes_item.matter.title)
+            ).strip()
         # matter.result_status is allowed to be null
         # only when no votes or Legistar EventItemMatterStatus is null
+        ev_minutes_item.minutes_item.description = rx.sub(
+            " ", str(ev_minutes_item.minutes_item.description)
+        ).strip()
+        ev_minutes_item.minutes_item.name = rx.sub(
+            " ", str(ev_minutes_item.minutes_item.name)
+        ).strip()
+
         if ev_minutes_item.matter and not ev_minutes_item.matter.result_status:
+            ev_minutes_item.matter.title = rx.sub(
+                " ", str(ev_minutes_item.matter.title)
+            ).strip()
```

Comment:
This set of changes, which is simplifying whitespace, is handled in https://github.com/CouncilDataProject/cdp-scrapers/pull/17 . Very sorry for not communicating that earlier.

Also, this method `fix_event_minutes()` is just moving the attributes around. The cleanup has already been done when we make the models like in `get_minutes_item()`, `get_matter()`, using `stripped()` (which will be called `str_simplified()` after https://github.com/CouncilDataProject/cdp-scrapers/pull/17 is merged).

Reply:
aah..ok. I didn;t know that. thought you are working on unicode string cleanuos and I tried to fix the rest of other string cleanups.  I will revert back.